src/society_stage/StrategicCamera.cs

```diff
@@ -137,6 +138,35 @@ public bool PanCamera(float delta, float forwardMovement, float leftRightMovemen
         return true;
     }
 
+    [RunOnKey("g_pan_mouse", CallbackRequiresElapsedTime = false)]
+    public bool PanCameraWithMouse(float delta)
```

Comment:
```suggestion
    public bool PanCameraWithMouse(float dummy)
```

I'd name the parameter like that to ensure no one later accidentally tries to use it and gets surprised when the variable doesn't actually contain the delta.